nox/sessions.py

```diff
@@ -184,6 +185,13 @@ def create_tmp(self) -> str:
         self.env["TMPDIR"] = tmpdir
         return tmpdir
 
+    @property
+    def cache_dir(self) -> pathlib.Path:
+        """Create and return a 'shared cache' directory to be used across sessions."""
+        path = os.path.join(self._runner.envdir, ".shared")
```

Comment:
While I'm poking around here I'd suggest sticking with `pathlib.Path` throughout

So I'd recommend changing this to:

```python
path = pathlib.Path(self._runner.envdir).joinpath(".shared")
```